model/host/volume.go

```diff
@@ -18,6 +19,10 @@ type Volume struct {
 	Host             string    `bson:"host" json:"host"`
 }
 
+type VolumeModifyOptions struct {
```

Comment:
I'm not sure why this is in the model package. also needs tags

Reply:
I agree a lot of these decisions were rough but I pretty much just followed the layout of HostModifyOptions and was going to go from there :) I can reorganize though